Vagrantfile

```diff
@@ -63,6 +64,7 @@ Vagrant.configure(2) do |config|
         # testing out changes locally.
         machine.vm.synced_folder dir, '/tmp/salt-testing-root/saltfs'
         machine.vm.synced_folder test_pillars_path, '/srv/pillar'
+        machine.vm.synced_folder test_wpt_path, '/srv/web-platform-tests'
```

Comment:
Why do this? Seems it would be easier to keep this with the rest of the app's files.